doc/articles/create-an-app-rider.md

```diff
@@ -37,6 +40,8 @@ Creating an Uno Platform project is done [using dotnet new](xref:Uno.GetStarted.
 
 > [!TIP]
 > If you are not able to run the online Live Wizard, you can explore the [`dotnet new` template](xref:Uno.GetStarted.dotnet-new) directly in the CLI.
+> [!IMPORTANT]
+> A notification window will appear for the free registration of the extension. If the notification disappears before you can enter the license, you can find it again in the "Notification Bell" at the top right of the Rider window.
```

Comment:
@jeromelaban I am wondering of the addition of this, since we don't mention anything for the registration for VS Code currently in the doc

Reply:
Sure, we could add that one as well. Will do.